docker/README.md

```diff
@@ -0,0 +1,95 @@
+# Docker container description for Tendermint applications
+
+* Overview (#Overview)
+* Tendermint (#Tendermint)
+* Basecoin (#Basecoin)
+* Ethermint (#Ethermint)
+
+## Overview
```

Comment:
Yes, that's one of the next steps I'm working on, I just didn't get there yet. :)